go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/crossplane/provider-aws
 go 1.13
 
 require (
+	github.com/aws/aws-sdk-go v1.15.78
```

Comment:
https://github.com/aws/aws-sdk-go/blob/ae9b9fd92af132cfd8d879809d8611825ba135f4/CHANGELOG.md#release-v1210-2019-07-18

1.21.0 introduce AWS web identity provider

Reply:
Currently, this pod is deployed in shoot, we just need ec2medata function.